README.md

```diff
@@ -36,6 +58,23 @@ Or more formally. You win as soon as you fulfill one of these conditions:
 - Capture all (4) capture points (the squares)
 - Eliminate all enemy units
 
+### Explanations
+
+| Empty | Player 2 won          |
+|---------------------------|----------------------------------|
+<img src="./public/images/board.png" alt="emptyboard" width="200"/> | <img src="./public/images/winByCapture.png" alt="winByCapture" width="200"/> |
+
+In this Example the red player won by capturing the obejctives.
+
+| Key | Functionality          |
+|---------------------------|----------------------------------|
+| <img src="./public/images/triangle.png" alt="triangle" width="50" align="left"/> | The triangle stands for a minor spawn point (produces units with simple power). From there you can create new units. |
+| <img src="./public/images/star.png" alt="star" width="50" align="left"/>  | The star stands for a major spawn point (produces units with double power). From there you can create new units. | 
+<img src="./public/images/square.png" alt="square" width="50" align="left"/> | The square stands for an objective to cover. |
+<img src="./public/images/simplePower.png" alt="simplePower" width="50" align="left"/> | This unit has a power of 1. | 
```

Comment:
Basically, a tile contains an army, and the number on the tile represents the number of units in that army. I think this is a better mental model then that of a single unit with more power.